tests/integration/test_cli.py

```diff
@@ -295,3 +295,67 @@ def test_stemhist(args, tmpdir):
             g = args[args.index('-g') + 1]
             exp_suffix = '.' + g + exp_suffix
         assert differ([path + exp_suffix, out + suffix, '-l', '1'])
+
+
+@pytest.mark.parametrize("path", FONTS)
+def test_outpath_intuitive(path, tmpdir):
```

Comment:
```suggestion
def test_outpath_order(path, tmpdir):
```

Reply:
Changed this and a couple of others (replacing "intuitive" with "order").